openapi.yaml

```diff
@@ -139,54 +139,72 @@ info:
     
     openEO-based back-ends are usually hosted on a different domain / host than the client that is requesting data from the back-end. Therefore most requests to the back-end are blocked by all modern browsers. This leads to the problem that the JavaScript library and any browser-based application can't access back-ends. Therefore, all back-end providers SHOULD support CORS to enable browser-based applications to access back-ends. [CORS is a recommendation of the W3C organization](https://www.w3.org/TR/cors/). The following chapters will explain how back-end providers can implement CORS support.
     
-    ## OPTIONS method
+    ## CORS headers
```

Comment:
Maybe change title to "General CORS Headers" to make it easier to refer to this from the OPTIONS section

Reply:
Hmm, I'd rather not change it to not break existing links to this header.